src/js/version.ts

```diff
@@ -1,3 +1,3 @@
 export const SDK_PACKAGE_NAME = 'npm:@sentry/react-native';
 export const SDK_NAME = 'sentry.javascript.react-native';
-export const SDK_VERSION = '4.15.0';
+export const SDK_VERSION = '5.0.0-rc.1';
```

Comment:
Should be `5.0.0` now?

Reply:
The release CI pipeline will update the version.